Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -581,28 +581,136 @@ def panorama_commit_command(args: dict):
     """
     Commit and show message in the war room
     """
-    result = panorama_commit(args)
+    use_polling = args.get('polling', 'false') == 'true'
+    interval_in_seconds = int(args.get('interval', '60'))
+    timeout = int(args.get('timeout', '600'))
+    job_id = args.get('job_id')
+    script_results = []
+
+    # Support polling
+    if use_polling:
```

Comment:
Theres a decorator called `polling_function` in CommonServerPython that Id reccomend using if possible

Reply:
Do you have any documentation on how this shoudl be used? I could not find it in the devdocs.

Reply:
@aburt-demisto currently there isn't, the best thing to do is to look at it has been used in integrations so far, maybe that could bring you some insights

Reply:
also take a look at the docstrings there